examples/virtual-device-2-aws-connection/main.tf

```diff
@@ -0,0 +1,99 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+provider "aws" {
+  access_key = var.additional_info[0]["value"]
+  secret_key = var.additional_info[1]["value"]
+  region     = var.zside_seller_region
+}
+
+#Network Edge Module
```

Comment:
The comment is misleading. We aren't using any Network Edge Modules. We're using network edge resources.